integration/integration_api_test.go

```diff
@@ -0,0 +1,58 @@
+// +build integration
+// can be execute with go test -tags=integration ./integration/...
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/piperenv"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPiperGithubPublishRelease(t *testing.T) {
+	t.Parallel()
+	token := os.Getenv("PIPER_INTEGRATION_GITHUB_TOKEN")
+	if len(token) == 0 {
+		t.Fatal("No GitHub token maintained")
+	}
+
+	owner := os.Getenv("PIPER_INTEGRATION_GITHUB_OWNER")
+	if len(owner) == 0 {
+		owner = "OliverNocon"
+	}
+	repository := os.Getenv("PIPER_INTEGRATION_GITHUB_REPOSITORY")
+	if len(repository) == 0 {
+		repository = "piper-integration"
```

Comment:
Can you provide more details about that repo: I know about `SAP/piper-validation`, but I don't know about a repo `piper-integration`.

I also would expect something like "&lt;org&gt;/&lt;repo&gt;" rather than simply "&lt;repo&gt;".

Reply:
it is just a test repo with permissions. This can easily be exchanged at some point.
For this test a repo is required where releases can be created without any impact ...